readthedocs/rtd_tests/tests/test_project_forms.py

```diff
@@ -230,6 +230,15 @@ def setUp(self):
             identifier='public/4',
             verbose_name='public/4',
         )
+        get(
+            Version,
+            project=self.project,
+            slug='stable',
+            active=True,
+            privacy_level=PUBLIC,
+            identifier='ab96cbff71a8f40a4240aaf9d12e6c10',
+            verbose_name='stable',
+        )
```

Comment:
Creating a version named `stable` is a specific test, we shouldn't merge it with the rest.

Reply:
@stsewd Okay :+1: . Updated the PR